src/common-model/entities.ts

```diff
@@ -141,11 +143,30 @@ export enum FollowupFieldsEnum {
   interval_of_followup = 'interval_of_followup',
 }
 
+export enum ExposureFieldsEnum {
```

Comment:
we should only add fields that are used in the data validation, since migration will be using this to check fields used in code

Reply:
👍